core/logic/src/main/java/org/apache/syncope/core/logic/SyncopeLogic.java

```diff
@@ -448,7 +448,7 @@ public NumbersInfo numbers() {
         numbersInfo.getConfCompleteness().put(
                 NumbersInfo.ConfItem.NOTIFICATION.name(), !notificationDAO.findAll().isEmpty());
         numbersInfo.getConfCompleteness().put(
-                NumbersInfo.ConfItem.PULL_TASK.name(), !taskDAO.findAll(TaskType.PULL).isEmpty());
```

Comment:
Did you remove the `TaskType` enum completely, right?

Reply:
Nope, going to remove it.

Reply:
We cannot remove `TaskType` enum, it is necessary for search and to build `TaskQuery` objects.